src/components/Card/CardHeader/CardHeader.props.ts

```diff
@@ -0,0 +1,21 @@
+import type { IconProps } from '@virtuslab/tetrisly-icons';
+
+import type { CardHeaderConfig } from '../Card.styles';
+
+import type { AvatarProps } from '@/components/Avatar';
+import type { BareButtonProps } from '@/components/Button/Button.props';
+
+export type CardHeaderProps = {
+  title: string;
+  description?: string;
+  beforeComponent?: BeforeComponentProps;
+  styles: CardHeaderConfig;
+  actions?: BareButtonProps[];
```

Comment:
We shouldn't allow to pass all possible props here, there should be limitations with size.

Reply:
okej

Reply:
temporarily on hold